api/wasm.go

```diff
@@ -84,6 +85,10 @@ const (
 	ValueTypeF32 ValueType = 0x7d
 	// ValueTypeF64 is a 64-bit floating point number.
 	ValueTypeF64 ValueType = 0x7c
+	// ValueTypeVector is a 128-bit vector value.
+	//
+	// TODO
```

Comment:
maybe take from here? Might have to fix wrapping


```suggestion
	// The type corresponds to a 128 bit vector of packed integer or floating-point data.
	// The packed data can be interpreted as signed or unsigned integers, single or double
	// precision floating-point values, or a single 128 bit type. The interpretation is
	//  determined by individual operations.
	//
	// See https://www.w3.org/TR/2022/WD-wasm-core-2-20220419/syntax/types.html#syntax-vectype
```

Reply:
also should mention that this occupies 2 uint64 and due to little endian encoding the lower 64 bits are first